toolbox/swiftsim/particle_splitting/generate_splitting_information.py

```diff
@@ -67,6 +67,38 @@ def load_hbt_config(config_path):
 
     return config
 
+def load_overflow_data(path_to_split_log_files):
+    '''
+    Loads particle splitting information for particles which
+    split enough times to overflow the SplitTrees dataset.
+
+    Parameters
+    ----------
+    config_path : str
+        Path to the HBT configuration used to analyse a simulation.
+
+    Returns
+    -------
+    overflow_data : dict
+        Dictionary indexed by (count, prog_id), where count was the split count
+        when the SplitTrees dataset was reset, and prog_id is the particle id that
+        forms the root of the new split tree.
+    '''
+    if path_to_split_log_files is None:
+        return None
+
+    overflow_data = {}
+    for filename in os.listdir(path_to_split_log_files):
```

Comment:
Are we guaranteed that there will be no other files/directories in this directory?

Reply:
No, but we don't know what the file name will be, so I don't think we can do a pattern match

Reply:
Isn't it hardcoded in swift or specified in the parameter file? Happy to leave as it is though

Reply:
I was wrong, turns out it is hardcoded. I will update this

Reply:
https://github.com/SWIFTSIM/HBTplus/pull/61/commits/71271ac31ed840c75a2825f51788bf8f2ad3309f